basics.py

```diff
@@ -0,0 +1,115 @@
+import pandas as pd 
+import matplotlib.pyplot as plt
+import numpy as np
+from pandas_datareader import data as web
+from scipy import stats
+import seaborn as sns
+from datetime import datetime
+
+pd.options.display.float_format = '{:.5f}'.format
+
+def get_data_close(symbol, start, end):
+    df = pd.DataFrame(web.DataReader(symbol, data_source='yahoo', start = start, end = end)['Adj Close'])
+    return df
+
+def changeCol(df, col, name):
+    df = df.rename(columns = {col : name})
+    return df
+
+def norm_dist(symbol, start,end =  datetime.today().strftime('%Y-%m-%d'),plot = False):
+    df = get_data_close(symbol, start, end)
+    df =changeCol(df, "Adj Close","Noarmallized {}".format(symbol))
+    #Normalizing the Ditribution
+    df =(df)/df.iloc[0]-1
+    if plot:
+        df.plot()
+        plt.show()
+    return df
+
+def daily_return(symbol, start,end =  datetime.today().strftime('%Y-%m-%d'), plot= False):
+    df_exp = get_data_close(symbol, start, end)
+    df_exp.dropna(inplace = True)
+    df_return = df_exp.pct_change()
+    df_return.iloc[0] = 0
+    df_return = changeCol(df_return,"Adj Close", "Daily Returns {}".format(symbol))
+    print(df_return.head())
+    if plot:
+        df_return['Daily Return'] = df_return['Daily Return']
+        df_return['Daily Return'].plot(title = "Daily Returns")
+        plt.show()
+    return df_return
+
+def cumulative_return(symbol, start,end =  datetime.today().strftime('%Y-%m-%d'),plot=False):
+    df_exp = get_data_close(symbol,start,end)
```

Comment:
Since we'll be having more maintainers in the future, it pays to have more descriptive variables. Can we change these right now?

Reply:
I will change the variable names to more meaningful ones.